compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/BUILD.bazel

```diff
@@ -87,6 +88,7 @@ iree_compiler_cc_library(
         "@llvm-project//mlir:Support",
         "@llvm-project//mlir:TensorDialect",
         "@llvm-project//mlir:VectorDialect",
+        "@llvm-project//mlir:VectorInterfaces",
```

Comment:
wait, we have `VectorInterfaces`?

Reply:
Yeah, there's a `VectorUnrollOpInterface` but it doesn't seem to do much though (just provides `getShapeForUnroll`)